### PR TITLE
PLT-4097 Stopped automatically focusing user list search box on mobile

### PR DESCRIPTION
**Pull request by hmhealey**
The search being focused when it opens seems to be messing up the size calculations for the modal on Android. 

@asaadmahmood Once this goes in (or I get my spinmint running), can you test this on your phone? I don't have an Android device yet so I can't see if it works there.

#### Ticket Link
https://mattermost.atlassian.net/browse/PLT-4097

#### Checklist
- Has UI changes

### CODE REVIEW

File: webapp/utils/user_agent.jsx [hunk 1]
```diff
@@ -81,6 +81,10 @@ export function isMobileApp() {
     return isAndroid() || isIos();
 }
 
+export function isMobile() {
```
Comment (enahum) [inline webapp/utils/user_agent.jsx]: Dude whats the difference between this function and the above `isMobileApp`?

Comment (hmhealey) [inline webapp/utils/user_agent.jsx]: Oh, I'm not smart enough to understand my own function names apparently. I assumed `isMobileApp` wasn't checking for mobile browsers for some reason. I'll fix that.

**Comment by jwilander** [comment]
+1

**Comment by enahum** [comment]
+1

**Pull request merged**